packages/app/src/cli/services/app/config/link.test.ts

```diff
@@ -116,32 +116,31 @@ describe('link', () => {
       expect(remoteApp).toEqual(mockRemoteApp({developerPlatformClient}))
     })
   })
-})
```

Comment:
was this change intentional? seems like it changed the indentation of all tests

Reply:
Yes, most of the tests here were outside of the `describe` where they should belong. Sorry for the noise 😅